main.go

```diff
@@ -80,7 +178,7 @@ func getAllRepo(input string, appName string) ([]UpdatedRepo, error) {
 	return allRepos, nil
 }
 
-//Remove git and DockerHub
+// checkForGitAndDockerHub Remove git and DockerHub
```

Comment:
if adding comments for private stuff, still recommended to start the comment with the name